src/yup/schemas/string/string.schema.ts

```diff
@@ -101,53 +101,57 @@ export const stringSchemaFormat = (
   const label = title || capitalize(key);
 
   if (format === "date-time") {
-    const path = joinPath(description, "format.dateTime");
-    const message =
-      getError(path) || `${label} is an invalid date and time format`;
+    const path = joinPath(description, "format");
```

Comment:
It seems kinda strange that some of the keys need to be nested into format ie: `key.format.dateTime`. this should probably be changed or at minimum documented.

Reply:
Yeh, will add an issue to add it to documentation. Will review that format feature, i think you wouldnt need the key if there's a single format being applied, but been a while since i've looked at the code.